training/commandlineflags.cpp

```diff
@@ -144,6 +144,12 @@ void PrintCommandLineFlags() {
 void ParseCommandLineFlags(const char* usage,
                            int* argc, char*** argv,
                            const bool remove_flags) {
+  if (*argc == 1) {
+    tprintf("USAGE: %s\n", usage);
+    PrintCommandLineFlags();
+    exit(0);
```

Comment:
training/unicharset_extractor.cpp and training/wordlist2dawg.cpp use exit code 1.
I therefore suggest using exit(1) here, too.